frontend/src/metabase/visualizations/components/ChartTooltip/EChartsTooltip/EChartsTooltip.stories.tsx

```diff
@@ -35,7 +35,7 @@ export const LightTheme = {
     </SdkVisualizationWrapper>
   ),
   args: {
-    theme: defineMetabaseTheme({
```

Comment:
Could we leave `defineMetabaseTheme()` here as it is helpful for type inference? We can bring `defineMetabaseTheme` to `metabase/embedding-sdk`

Reply:
Good point,  I didn't realize that the theme type was also moved, https://github.com/metabase/metabase/pull/52020/commits/ee873881f9ad0dfb0c5693d4e145454cad3f2453